.github/workflows/pr.yml

```diff
@@ -6,7 +6,7 @@ name: lint, style, and tests
 on:
   pull_request:
     branches:
-      - main
+      - "*"
```

Comment:
Do we really want PRs to all branches to trigger tests? Wouldn't it be better to have a naming convention for development branches?